src/controllers/api.controller.ts

```diff
@@ -212,7 +215,7 @@ export const claimsFeed = async (
       INNER JOIN "Node" as n3 on e2."endNodeId" = n3.id
       INNER JOIN "Claim" as c on e."claimId" = c.id
       WHERE NOT (n1."entType" = 'CLAIM' and e.label = 'source')
-      ORDER BY c.id DESC
+      ORDER BY c."effectiveDate" DESC
```

Comment:
(I accidentally committed this directly into master, letting it ride along here into dev; we want to order by recent claims about things that happened recently, sometimes we import old stuff)